source/codegen/metadata/nirfmxlte/functions.py

```diff
@@ -2511,8 +2511,6 @@
             {
                 'direction': 'in',
                 'enum': 'PsschModulationType',
-                'grpc_field_number': '4',
```

Comment:
This is what is causing the source breaking changes. Maybe there was a hand-edit that was not applied to scrapigen?

Reply:
I think this the versions of these functions coming from scrapigen were intended to be overridden with the ones specified in `functions_addon.py` for both files in [this PR](https://github.com/ni/grpc-device/pull/1049/files).

I think the problem is the [`__init__.py` for these](https://github.com/ni/grpc-device/blob/main/source/codegen/metadata/nirfmxlte/__init__.py) don't use the `functions_addon.py` file. 

@RahulBenaka , I think the fix is to update the __init__.py for SpecAn and LTE to be like the [ni_scope one](https://github.com/ni/grpc-device/blob/main/source/codegen/metadata/niscope/__init__.py) for instance where it takes in the `functions_addon` file and updates the functions with `metadata['functions'].update(functions_override_metadata)`.

Reply:
This fix worked. I have updated the code for bluetooth and specan personality